src/index.html

```diff
@@ -113,77 +120,123 @@ <h1 class="bottom__title">Gallop Academy</h1>
   </menu>
 
   <main class="main">
-    <div class="container">
-      <section class="about-us" id="about-us">
-        <div class="about-us__texts">
-          <h3 class="about-us__title">About our school</h3>
+      <div class="about-us-wrapper">
```

Comment:
please follow the rules BEM
about-us - Block
about-us__texts - Element
what is it 'about-us-wrapper' ?